modules/odc-customizing-available-cluster-roles-using-developer-perspective.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// applications/projects/working-with-projects.adoc
+
+[id="odc-customizing-available-cluster-roles-using-developer-perspective_{context}"]
+= Customizing the available cluster roles using the Developer perspective
+
+You can customize and configure an existing project access roles name by editing the YAML code of `console.openshift.io/display-name` of your cluster.
```

Comment:
Small details:
* Does it make sense to note here again that you need to changing the `ClusterRole`? But we are in the right section, so it is maybe obvious.
* You should mention that this is an `annotation`, otherwise the admin could also think you mean a label.

```suggestion
You can customize and configure an existing project access roles name by editing the YAML code, and add an `console.openshift.io/display-name: Display name` annotation of your cluster.
```

Reply:
Got it. This suggestion seems fine I'll rephrase the sentence a bit to incorporate the changes.